src/database/standalone-sqlite.ts

```diff
@@ -880,6 +881,26 @@ export class StandaloneSqliteDatabaseWorker {
     });
   }
 
+  getTxByOffset(offset: number): TxByOffsetResult {
```

Comment:
The offset comparison logic is tricky enough to warrant unit tests. In particular, we should test the boundary conditions, offsets at the beginning and end of the transaction.

Reply:
I did test this on png images and they came out correct. I can see how this can be problematic if the chunk offset requested in the http request for falls on some boundary. From unit test point of view, to give that a fair test, I'll want to run sql query in the unit test, with various offsets.

Reply:
unit tests testing the boundry have been added https://github.com/ar-io/ar-io-node/pull/370/commits/bef4bce273c066275ef73463a2aa1e20dd638a67